.github/workflows/release.yaml

```diff
@@ -87,12 +87,13 @@ jobs:
           # Create the release notes for the Github release
           git config user.name rhobs-release-bot
           git config user.email release-bot@monitoring.rhobs.io
-          npx standard-version -i RELEASE_BODY.md --release-as $(cat VERSION) --skip.commit
+          npx standard-version -i RELEASE_BODY.md  \
+            --release-as $(cat VERSION) \
+            --skip.commit
 
       - name: Publish tag
         id: publish_tag
         run: |
-          git push --follow-tags
```

Comment:
So this step will just echo the tag name?

Reply:
git push --follow-tags will push the tag that got created in an earlier step

Reply:
Can we remove this step then?

Reply:
no, it is important to push the tag ... just like the old release workflow. 
What is the concern 🤔

Reply:
Updated. Thanks.